conda_forge_tick/make_graph.py

```diff
@@ -119,10 +119,14 @@ def populate_feedstock_attributes(
                 "bot",
             }
         }
-
-    yaml_dict = ChainDB(
-        *[parse_meta_yaml(meta_yaml, platform=plat) for plat in ["win", "osx", "linux"]]
-    )
+    build_subdirs = ALL_BUILD_SUBDIRS if 'provider' in sub_graph['conda-forge.yml'] else MAIN_SUBDIRS
```

Comment:
This check is not quite right as people can list provider in feedstocks but still have no aarch packages. Also, some packages do not build windows.

Reply:
Can you give a case where the provider is used but a package is not built? The windows case is handled below with a skip check

Reply:
Any feedstock with provider: win: azure.

Reply:
It's not so much if they exist but that they could exist and this would be totally fine.

Reply:
Ah ok, so I need to do a bit more thorough check of `provider`. We might consider adding a provider for each build_subdir for good measure.